packages/ui/src/components/BlockDetails.tsx

```diff
@@ -160,7 +156,7 @@ const blockAttrs: (block: BlockResult) => Array<[string, any]> = (
     ['Height', block.height],
     ['Era ID', block.eraId],
     ['State', block.state],
-    ['Block Hash', id],
+    ['Block Hash', <Link to={id}>{id}</Link>],
```

Comment:
I don't think the link works. You should construct the link address by yourself.

Reply:
The link works. I already checked it before push. Here id means blockhash

Reply:
Ok, you are right. It is because it is on the same page.